docs/issue_template.md

```diff
@@ -0,0 +1,37 @@
+<!-- This issue template is from: https://www.talater.com/open-source-templates/#/page/98 -->
+# Issue:
+<!--- Name your issue -->
+
+## Issue summary:
+<!--- Provide a general summary of the issue in the field above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what actually happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+<!--- You may include a reprex if you wish: https://github.com/tidyverse/reprex -->
+1.
+2.
+3.
+4.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
```

Comment:
Should we provide a command to run instead that would include most of this information?

Reply:
Yes that would be good, they could start with just running R.version(). We don't have "hydromad.version()", but we can simply them to report the version number

Reply:
`packageVersion` appears to be the generic function for getting an installed package version

Reply:
That's a good option. You might also consider asking users to list the [packages and/or namespaces](https://stats.idre.ucla.edu/r/faq/how-does-r-handle-overlapping-object-names/) they have in their environment using `(.packages())` or `loadedNamespaces()` so that you can have a better understanding of whether there could be something in their environment that is interfering with hydromad. This may be a somewhat unlikely scenario because hydromad seems to have fairly unique function names, but it's worth thinking about.

Reply:
Addressed by https://github.com/josephguillaume/hydromad/pull/50/commits/4ff22969dc0b267d08ac3009d2490240039dcff0